website/language.py

```diff
@@ -68,7 +68,11 @@
 
 CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
 
-CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a possible conflict with add-ons. Please deactivate any add-ons authorized on the account to be merged and try again.'
+CANNOT_MERGE_ACCOUNTS_LONG = (
+    'Accounts cannot be merged due to a possible conflict with add-ons.  '
+    'Before you continue, please <a href="/settings/addons/"> deactivate '
+    'any add-ons</a> authorized on this account and try again.'
```

Comment:
I think this language could be a bit clearer on which account it refers to, especially since users can see this message while they are not logged in to either account. Instead of "deactivate any add-ons authorized on this account", how about changing it to "deactivate any add-ons authorized on the account to be merged" or "to be merged into your primary account"?